components/prism-docker.js

```diff
@@ -0,0 +1,8 @@
+Prism.languages.docker = {
+	'keyword': {
+    pattern: /^\s*\b(ONBUILD|FROM|MAINTAINER|RUN|EXPOSE|ENV|ADD|COPY|VOLUME|USER|WORKDIR|CMD|ENTRYPOINT)\b\s/mi
+	},
+  'string': /("|')(\\\n|\\?.)*?\1/,
+	'comment': /#[^\n]+/,
```

Comment:
I think the `[^\n]+` could be replaced with `.+`. This would even add support for `\r`.